variables.tf

```diff
@@ -42,14 +53,7 @@ variable "instance_type" {
 
 variable "key_name" {
   description = "SSH key name in your AWS account for AWS instances."
-    default   = "amazonhosts"
-}
-
-variable "key_path" {
-  description  = "Path to the private key specified by key_name."
-    default    = {
-      key_path = "/Users/alaric/amazonhosts.pem"
-    }
+  default     = "amazonhosts"
```

Comment:
I know it is outside of this PR, but can we set this `default` value to something all AWS users have already (`""`) unless they provide their own value.

/cc @tfhartmann